docs/disable-if-parameter.adoc

```diff
@@ -0,0 +1,83 @@
+:page-title: Disable Parameterized Test Based on Parameter Values
+:page-description: Extends JUnit Jupiter with `@DisableIfParameter`, which selectively disables parameterized tests
+
+The `@DisableIfParameter` annotation can be used to selectively disable parameterized tests based on their parameter values (converted with `toString()`).
+The annotation is only supported on test method level for parameterized tests.
+Unlike the `@Disabled` API provided in JUnit Jupiter, which disables the test on first encounter of the annotation , `@DisableIfParameter` is validated before each parameterized test execution.
+As a consequence, instead of disabling the entire set of parameterized tests, each test can be evaluated and possibly disabled individually.
+
+[source,java]
+----
+// disable invocations whose parameter contains "she"
+@DisableIfParameter(contains = "she")
+@ParameterizedTest
+@ValueSource(strings = {
+        "Tread lightly, she is near",
+        "Under the snow,",
+        "Speak gently, she can hear",
+        "The daisies grow."
+})
+void interceptContains(String line) {
+}
+----
+
+The test `interceptContains` ordinarily would run four times, but because the first and third lines contain the word "she", those invocations get disabled.
+
+You can specify more than one substring at a time:
+
+[source,java]
+----
+@DisableIfParameter(contains = { "bright", "dust" })
+@ParameterizedTest
+@CsvSource(delimiter = ';', value = {
+        "All her bright golden hair;Tarnished with rust,",
+        "She that was young and fair;Fallen to dust."
+})
+void interceptContainsAny(String line, String line2) {
+}
+----
+
+`DisableIfParameter` will disable the test invocation if any parameter value matches any value from `contains`.
+In this case, the test `interceptContainsAny` should have run two times, but both invocations got disabled.
+The first invocation because the first parameter contains "bright".
+The second invocation because the second parameter contains "dust".
```

Comment:
```suggestion
The first invocation got disabled, because the first parameter contains "bright".
The second execution got skipped, because the other parameter contains "dust".
```